envycontrol.py

```diff
@@ -207,6 +217,15 @@ def graphics_mode_switcher(graphics_mode, user_display_manager, enable_force_com
             f"Enable PCI-Express Runtime D3 (RTD3) Power Management: {rtd3_value or False}")
         cleanup()
 
+        if logging.getLogger().level == logging.DEBUG:
+            service = subprocess.run(["systemctl", "enable", "nvidia-persistenced.service"])
```

Comment:
What is the purpose of using nvidia-persistenced? It doesn't bring any improvements for PRIME setups. Nvidia-persistenced only makes sense for headless computational graphics setups, i.e. without a running X session. This does not affect the operation of RTD3 in any way.

Reply:
Before adding the inclusion of nvidia-persistent I ran some tests and without this service enabled the total system consumption was higher

Reply:
I saw no change on my system with AMD iGPU + 3050 Mobile when enabling nvidia-persistenced. Can you provide steps to reproduce the powerdown along with nvidia-persistenced? Just FYI that nvidia-persistenced, according to the source code, does nothing related to GPU power managment other than disabling NUMA memory blocks used by CUDA.

Reply:
Maybe in your case nothing has changed since you have amd I have done tests on Nvidia + Intel bundle and with the service enabled the total system consumption which I looked through upower was lower by several watts to make sure that it is not a trivial error I have done these tests several times and the consumption was always lower with this service than without it.